index.js

```diff
@@ -211,6 +215,16 @@ contact.addEventListener('click', () => {
   navMobile.classList.add('hidden');
 });
 
+submitBtn.addEventListener('click', (e) => {
+  if (re.test(email.value) && email.value === email.value.toLowerCase()) {
+    errMsgEmail.innerText = '';
+  } else {
+    errMsgEmail.innerText = 'Your email should be in lowercase!';
+    errMsgEmail.classList.add('error-message');
+    e.preventDefault();
+  }
```

Comment:
- [ ] Good job on creating the error message when the email is not in lowercase 👏 However, when the inputs are correct the form is not sent normally 👇 Kindly fix this issue 💯 

![Animation](https://user-images.githubusercontent.com/50721479/216363696-500b344d-b4a2-4e27-a342-2fe1a1c0fe72.gif)